docs/source/features/reasoning_outputs.md

```diff
@@ -288,32 +287,6 @@ class ExampleParser(ReasoningParser):
         """
 ```
 
-Additionally, to enable structured output, you'll need to create a new `Reasoner` similar to the one in <gh-file:vllm/model_executor/guided_decoding/reasoner/deepseek_reasoner.py>.
```

Comment:
Can you change this into vllm/reasoning/deepseek_r1_reasoning_parser.py instead?



Thanks

Reply:
hi, updated, PTAL

Reply:
Just need to change L291, Please keep the example here.